tests/unit/mixins/data-adapter-mixin-test.js

```diff
@@ -53,10 +54,14 @@ describe('DataAdapterMixin', () => {
     it('asserts `authorize` is overridden', function() {
       adapter.set('authorizer', null);
 
-      expect(function() {
-        adapter.ajaxOptions();
-        hash.beforeSend();
-      }).to.throw(/Assertion Failed/);
+      let [ major_version ] = DS.VERSION.split(".");
+
+      if (major_version < 2) {
+        expect(function() {
+          adapter.ajaxOptions();
+          hash.beforeSend();
+        }).to.throw(/Assertion Failed/);
+      }
```

Comment:
I think the whole test needs to be wrapped as otherwise it will fail as there are no assertions.

Reply:
It wasn't failing, maybe `mocha` doesn't have that issue? I moved the `if` outside anyway since it doesn't make sense to have an empty test.

Reply:
Hm, yeah - maybe I mixed that up with QUnit's behavior.